recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc.sh

```diff
@@ -53,6 +53,10 @@ tdx_enc_exit_error() {
 tdx_enc_prepare_generic() {
     tdx_enc_log "Preparing and checking system (generic)..."
 
+    if ! modprobe dm-crypt; then
```

Comment:
There's type on the commit message introducing this change: "inicialization"

Reply:
Good catch! Commit message updated.